### PR TITLE
Modify tests to use new mongomock API

### PR DESCRIPTION
**Pull request by mgalka**
KernelCI Backend MongoDB has been update, hence PyMongo has been
upgraded to version 3.9. This commit upgrades mongomock to version 3.18
to support PyMongo API changes.

Signed-off-by: Michal Galka <michal.galka@collabora.com>

### CODE REVIEW

**Comment by gctucker** [comment]
@mgalka Nice :)  So, can we re-enable some unit tests in Travis CI now?

**Comment by mgalka** [comment]
> @mgalka Nice :) So, can we re-enable some unit tests in Travis CI now?

@gctucker 
Re-enabled following test cases in Travis Ci:
-    utils/tests
-    utils/build/tests
-    handlers/common/tests

File: app/utils/boot/tests/test_boot_import.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ class TestParseBoot(unittest.TestCase):
 
     def setUp(self):
         logging.disable(logging.CRITICAL)
-        self.db = mongomock.Database(mongomock.Connection(), "kernel-ci")
+        self.db = mongomock.Database(mongomock.MongoClient(),
+                                     "kernel-ci", None)
```
Comment (gctucker) [inline app/utils/boot/tests/test_boot_import.py]: Ha, it's a shame there's no `_store=None` default value, but that's how the API is:
https://github.com/mongomock/mongomock/blob/develop/mongomock/database.py#L23

Comment (mgalka) [reply app/utils/boot/tests/test_boot_import.py]: Absolutely, that was actually my first thought

**Pull request merged**